### DISCUSSION TITLE
Is this possible?

### DISCUSSION THREAD

**Discussion by McCugor** [microsoft-search/pnp-modern-search#4150] [Q&A] [answered]
Hello,

New to PNP Search and concepts such as Handles Bars. Hoping someone would be kind enough to provide some advice.

I have setup a simple PNP Search web part as below. The webpart references a SharePiont Document Library, the columns as mapped properties.

I would like the "Next Review" date to be conditionally formated, to turn red if it is equal to or lesser than the current date. 

Is this achievable and how would I begin to approach this?

<img width="647" alt="WebpartSettings2" src="https://github.com/user-attachments/assets/c0af7c8e-043f-4b5f-a823-4fa686b8e7f3" />
<img width="233" alt="WebpartSettings1" src="https://github.com/user-attachments/assets/8a227820-35a1-4ea8-9380-6ff8951c7e5a" />
<img width="601" alt="WebpartDatetoChange" src="https://github.com/user-attachments/assets/0d34fad6-809e-478e-97cd-62c709550ceb" />

**Comment by kasperbolarsen** [comment, marked as answer]
Absolutely
based on https://github.com/microsoft-search/pnp-modern-search/discussions/3697

I got this to work by setting the handlebar of the date field to (the css needs to be updated and the colors should be as per the Theme):


`{{#if (moment (slot item @root.slots.TheDate) isBefore=(moment))}}
    <div class="datesample" style="background-color:red">
{{else}}
    <div class="datesample" style="background-color:green">
{{/if}}
{{getDate (slot item @root.slots.TheDate) 'LL'}}    
</span>`


![image](https://github.com/user-attachments/assets/817de898-dcc4-4b28-b870-f3fe41893b9a)

**Reply by McCugor** [reply]
Thats works great. Removed some of the code because I just needed the text to highlight.

Thank you so much. 

<img width="606" alt="WebpartDatetoChange2" src="https://github.com/user-attachments/assets/d3ee3a01-ef82-4f22-bb66-f82cf78b686f" />
